jjb/dynamic/baseline.yml

```diff
@@ -0,0 +1,158 @@
+- job:
+    block-downstream: false
+    block-upstream: false
+    builders:
+    - shell: |-
+        set -e
+
+        if [[ ! -d "${WORKSPACE}/perf-dept" ]]; then
+        git clone https://${GITHUB_TOKEN}@github.com/redhat-performance/perf-dept.git
+        fi
+        export PRIVATE_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf
+        chmod 600 ${PRIVATE_KEY}
+        export OPTIONS="-o StrictHostKeyChecking=no -o ServerAliveInterval=1 -o ConnectionAttempts=100"
+        export KUBECONFIG="/root/.kube/config"
+
+
+        if [[ "${USE_PROXY}" == "true" ]]; then
+        ssh ${OPTIONS} -o ProxyCommand="ssh -i ${PRIVATE_KEY} -W %h:%p ${PROXY_USER}@${PROXY_HOST}" -i ${PRIVATE_KEY} ${USER}@${JUMP_HOST} 'bash -s' <<ENDSSH
```

Comment:
Ansible should "proxy" for us since we can set to localhost or a remote host.

Reply:
Agreed, fixed it.